homeassistant/components/tts/baidu.py

```diff
@@ -0,0 +1,111 @@
+"""
+Support for the baidu speech service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tts.baidu/
+"""
+
+import asyncio
+import logging
+import voluptuous as vol
+
+from homeassistant.components.tts import Provider, PLATFORM_SCHEMA, CONF_LANG
+import homeassistant.helpers.config_validation as cv
+
+
+REQUIREMENTS = ["baidu-aip==1.6.6"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+SUPPORT_LANGUAGES = [
+    'zh',
+]
+DEFAULT_LANG = 'zh'
+
+
+CONF_APPID = 'appid'
+CONF_APIKEY = 'apikey'
```

Comment:
Use CONST_API_KEY from https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py#L31
Try to use the constants from const.py to make the component consistent with other components

Reply:
I changed it.